folly/container/detail/F14Mask.h

```diff
@@ -46,7 +47,11 @@ FOLLY_ALWAYS_INLINE static unsigned findFirstSetNonZero(T mask) {
 using MaskType = uint64_t;
 
 constexpr unsigned kMaskSpacing = 4;
-#else // SSE2
+#elif FOLLY_SSE >= 2
+using MaskType = uint32_t;
+
+constexpr unsigned kMaskSpacing = 1;
+#else
```

Comment:
What's the purpose of this extra condition here? It looks to be identical to what's already here.

Reply:
Will remove.